cmd/helm/helm.go

```diff
@@ -67,10 +67,16 @@ func main() {
 	actionConfig := new(action.Configuration)
 	cmd := newRootCmd(actionConfig, os.Stdout, os.Args[1:])
 
-	if err := actionConfig.Init(settings.RESTClientGetter(), settings.Namespace(), os.Getenv("HELM_DRIVER"), debug); err != nil {
-		debug("%+v", err)
+	subCmd, _, err := cmd.Find(os.Args[1:])
+	if err != nil {
 		os.Exit(1)
 	}
+	actionConfigRequired := action.ConfigRequired(subCmd.Use)
```

Comment:
I think you should use `subCmd.Name()` instead of `subCmd.Use`.  I believe it will also fix the crash.